tss-esapi/tests/integration_tests/structures_tests/lists_tests/pcr_selection_list_tests.rs

```diff
@@ -91,35 +99,43 @@ fn from_tpml_retains_order() {
 
 #[test]
 fn test_subtract() {
```

Comment:
I'm not sure I follow what's being tested here - is it the subtraction alone or the conversion for `PcrSelectionList` as well?

Reply:
I am not really sure why the code is written like that. It is like if the builder didn't exists when the code was being written so all the lists are created from their TPM counterparts.